modules/util.rb

```diff
@@ -71,4 +71,22 @@ def self.full_avatar(user)
       event.respond event.channel.slowmode_rate
     end
   end
+
+  command :voicekick, {
+    help_available: true,
+    usage: '.voicekick [user] [channel]',
+    min_args: 1,
+    max_args: 2,
+    required_permissions: %i[manage_users]
```

Comment:
unfortunately, discordrb [only supports setting one, global string](https://github.com/shardlab/discordrb/blob/43da7657731d2da7ff68c332f6967b9a0f158476/lib/discordrb/commands/command_bot.rb#L243) for this feature, which means localization does not work.

so permission checks usually happen manually ([example](https://github.com/arch-community/qbot/blob/7b217bfc059fa81d06bb6184d30e7b52e77afbf7/modules/colors.rb#L143)).